templates/textfield.php

```diff
@@ -1,8 +1,16 @@
+<?php
+/**
+ * Template file for a textfield
```

Comment:
`text field` is two words, I think (e.g. `date field`). 

That said, we already have something of a style for template file headers in `master` in the [`Datepicker`](https://github.com/alleyinteractive/wordpress-fieldmanager/blob/master/templates/datepicker.php) and [`Checkboxes`](https://github.com/alleyinteractive/wordpress-fieldmanager/blob/master/templates/options-checkboxes.php) templates.

I don't know whether it matters what the style is, but I think we could try to be consistent.